pyscitt/pyscitt/client.py

```diff
@@ -231,7 +231,7 @@ def __init__(
             A bearer token for all requests made by this instance.
 
         member_auth:
-            MemberAuthenticationMethod include A pair of certificate and private key in PEM format or AKV login identity, used to sign requests.
+            MemberAuthenticationMethod include A pair of certificate and private key in PEM format, used to sign requests.
```

Comment:
Dropping this seems fine, but could probably be a separate PR without drawback and with a slight clarity benefit?